docs/serverless/08-07-sync-function-with-gitops.md

```diff
@@ -0,0 +1,252 @@
+
+1. Install flux:
+
+  ```bash
+  brew install fluxctl
+  ```
+
+2. Create a `flux` Namespace for the Flux operator's CRDs.
+
+  ```bash
+  kubectl create namespace flux
+  ```
+
+3. Export details of your GitHub repository - its name, the account name and related e-mail address, and the folder to which you will push Function and Trigger CRs built from local sources. Flux will synchronize the cluster with the content of this folder on the `main` branch.
+
+  ```bash
+  export GH_USER="{USERNAME}"
+  export GH_REPO="{REPOSITORY_NAME}"
+  export GH_EMAIL="{EMAIL_OF_YOUR_GH_ACCOUNT}"
+  export GH_FOLDER="{GH_FOLDER_FOR_FUNCTION_RESOURCES}"
+  ```
+
+4. Run this command to apply CRDs of the Flux operator to the `flux` Namespace on your cluster.
+
+  ```bash
+  fluxctl install \
+  --git-user=${GH_USER} \
+  --git-email=${GH_EMAIL} \
+  --git-url=git@github.com:${GH_USER}/${GH_REPO}.git \
+  --git-path=${GH_FOLDER} \
+  --namespace=flux | kubectl apply -f -
+  ```
+
+  You will see that Flux created these CRDs:
+
+  ```bash
+  serviceaccount/flux created
+  clusterrole.rbac.authorization.k8s.io/flux created
+  clusterrolebinding.rbac.authorization.k8s.io/flux created
+  deployment.apps/flux created
+  secret/flux-git-deploy created
+  deployment.apps/memcached created
+  service/memcached created
+  ```
+
+5. Obtain the certificate (SSH key) that Flux generated:
+
+  ```bash
+  fluxctl identity --k8s-fwd-ns flux
+  ```
+
+6. Run this command to copy the SSH key to the clipboard:
+
+  ```bash
+  fluxctl identity --k8s-fwd-ns flux | pbcopy
+  ```
+
+7. Go to **Settings** in your GitHub account:
+
+  ![GitHub account settings](./assets/settings.png)
+
+8. Go to the **SSH and GPG keys** section and select the **New SSH key** button.
+
+  ![Create a new SSH key](./assets/create-ssh-key.png)
+
+9. Provide the new key name, paste the previously copied SSH key, and confirm changes by selecting the **Add SSH Key** button.
+
+  ![Add a new SSH key](./assets/add-ssh-key.png)
+
+### Create a Function
+
+Now that Flux is authenticated to pull changes from your Git repository, you can start creating CRs from your local workspace files.
+You will create a sample inline Function and modify it by adding a trigger to it.
+
+1. Back in the terminal, clone this GitHub repository to your current workspace location:
+
+  ```bash
+  git clone git@github.com:${GH_USER}/${GH_REPO}.git
+  ```
+
+2. Go to the repository folder:
+
+  ```bash
+  cd {LOCAL_GITHUB_REPOSITORY_FOLDER}
+  ```
+
+3. Run the `apply` Kyma CLI command to create a Function CR in the YAML format on your remote GitHub repository. This command will generate the output in the `my-function.yaml` file.
+
+  ```bash
+  kyma apply function --filename {FULL_PATH_TO_LOCAL_WORKSPACE_FOLDER}/config.yaml --output yaml --dry-run > ./{GH_REPO}/${GH_FOLDER}/my-function.yaml
+  ```
+
+4. Push the local changes to the remote repository:
+
+  ```bash
+  git add .  # Stage changes for the commit
```

Comment:
Consider using a tab on these comments to move them to the right so as to make a straight column.